### PR TITLE
Ensures that rerunning the indexing process doesn't make duplicates (#217).

### PR DESCRIPTION
**Pull request by bwatson78**
- .rubocop.yml: ignores the test page for rubo errors.
- spec/services/oai_processing_service_spec.rb: adds reindexing test, as well as refactors repetitive code.
- spec/spec_helper.rb: adds match negater method.

### CODE REVIEW

File: spec/services/oai_processing_service_spec.rb [hunk 1]
```diff
@@ -25,10 +31,27 @@
...
+        end
+
+        it 'produces the same unique records when running the indexer on the same material' do
+          expect(number_of_docs).to eq 4
+          expect(response_ids).to match_array(ids_array)
+
+          process_indexer
+
+          expect(number_of_docs).to eq 4
+          expect(response_ids).to match_array(ids_array)
```
Comment (rotated8) [inline spec/services/oai_processing_service_spec.rb:52]: I was thinking of the [change matcher](https://relishapp.com/rspec/rspec-expectations/v/2-0/docs/matchers/expect-change#expecting-no-change) for this. Are `number_of_docs` and `response_ids` updated after the indexer is run, or are their values cached?

**Review by rotated8 — COMMENTED** [review]
One question, then g2g.

File: spec/services/oai_processing_service_spec.rb (hunk 1 above)
Comment (bwatson78) [reply spec/services/oai_processing_service_spec.rb:52]: You're right, the values are cached. Making changes.

Comment (bwatson78) [reply spec/services/oai_processing_service_spec.rb]: @rotated8 Ready for a re-review.

**Review by mlooney — APPROVED** [review]

**Pull request merged**